corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html

```diff
@@ -13,11 +13,18 @@
         }
     </style>
 {% endblock %}
+{% block js %}{{ block.super }}
```

Comment:
did you ever look into actually extending the reporting templates? seems ok if it was too hard to do that, just wanted to check if you had tried